package.json

```diff
@@ -37,11 +37,14 @@
       "at-rule-no-unknown": [
         true,
         {
-          "ignoreAtRules": [
-            "include",
-            "mixin",
-            "extend"
-          ]
+          "ignoreAtRules": ["include", "mixin", "extend"]
+        }
+      ],
+      "at-rule-empty-line-before": [
+        "always",
+        {
+          "ignore": ["after-comment"],
+          "ignoreAtRules": ["import", "include", "mixin", "extend"]
```

Comment:
I'm just curious, what do these do?

Reply:
[Read it here](https://stylelint.io/user-guide/rules/at-rule-empty-line-before/).

With this `ignoreAtRules` you can have `@rules` inside a css selector without having an empty line.

```css
.foo {
  color: red;
  
  @include positionX; // an extra empty line is needed
}

.foo-with-ignoreAtRules {
  color: red;
  @include positionX; // no extra empty line needed
}
```